src/reynir/Greynir.grammar

```diff
@@ -2284,16 +2296,30 @@ Sögn_2_nh →
 
 Andlög/fall/afall →
     NlÓbeintAndlag/fall AndlögBotn/fall/afall
+    | UmsnúinAndlög/fall/afall
 
 Andlög/fall/afall/tala/kyn →
     NlÓbeintAndlag/fall LoTengtSögn_nf/tala/kyn? AndlögBotn/fall/afall
+    | UmsnúinAndlög/fall/afall/tala/kyn
+
+UmsnúinAndlög/fall/afall →
+    NlBeintAndlag/afall UmsnúinAndlögBotn/fall/afall
```

Comment:
Since /afall is never used in `UmsnúinAndlögBotn`, it can (and should) be omitted, i.e. it should be `UmsnúinAndlögBotn/fall`.